### ISSUE TITLE
Analysis of what to Incorporate WCAG 2.2 errata into WCAG2ICT

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#580]
This issue is to track analysis of what changes might need to be made to WCAG2ICT due to the WCAG 2.2 update. Issue #607 is for tracking implementation.

The following list of issues are causing changes in WCAG 2.2. Need to determine if there will be any content changes that would need to be incorporated into the existing text in WCAG2ICT:

- [x] Check with @mbgower if this is the correct list of PRs that got incorporated into the republish of WCAG 2.2.

**Results of conversation with Michael** indicate that the easiest route to know the differences is to analyze a diff file between original WCAG 2.2 Recommendation and the December republish.

**Comment by maryjom** [comment]
@pday1 @ChrisLoiselle Mitch had a good idea that we do the analysis with this issue and the PR with changes on the other issue.

**Comment by maryjom** [comment]
@ChrisLoiselle @pday1 I asked Mike about the list of PRs incorporated into the latest WCAG, but I think it's harder to muddle through since PRs for Understanding docs and WCAG techniques are included in the list - they're all in the same repository.   Therefore I've created a diff version between the earlier WCAG 2.2 and the newly published WCAG 2.2 that we can more easily work through.  We can divide the work by thirds:
1. [ ] Definitions section
2. [ ] SC section (first half)
3. [ ] SC sections (second half)

**Comment by mbgower** [comment]
Mary Jo,
The below history of commits I sent you does not primarily show any understanding or technique updates. It seems to encompass changes to the spec only (with the exception of one or two merges that include both informative and normative, from when we were less disciplined).
You’ll note the batch that got merged on Nov 19, after the CFC. I think that is the primary set you’re looking for, but this history also contains other changes which were not deemed necessary to go through the CFC process, since they are on non-normative parts of the spec, etc.
https://github.com/w3c/wcag/commits/main/guidelines

Perhaps I’m not understanding what you’re looking for?

Mike

From: Mary Jo Mueller ***@***.***>
Date: Wednesday, January 22, 2025 at 11:51 AM
To: w3c/wcag2ict ***@***.***>
Cc: Michael Gower ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [w3c/wcag2ict] Analysis of what to Incorporate WCAG 2.2 errata into WCAG2ICT (Issue #580)
@ ChrisLoiselle @pday1 I asked Mike about the list of PRs incorporated into the latest WCAG, but I think it's harder to muddle through since PRs for Understanding docs and WCAG techniques are included in the list - they're all in the same repository. 


@ChrisLoiselle<https://github.com/ChrisLoiselle > @pday1<https://github.com/pday1 > I asked Mike about the list of PRs incorporated into the latest WCAG, but I think it's harder to muddle through since PRs for Understanding docs and WCAG techniques are included in the list - they're all in the same repository. Therefore I've created a diff version between the earlier WCAG 2.2 and the newly published WCAG 2.2 that we can more easily work through. We can divide the work by thirds:

  1.  Definitions section
  2.  SC section (first half)
  3.  SC sections (second half)

**Comment by maryjom** [comment]
I'll take the first half of the SCs and any general things I find and list them here:
- [ ] Everywhere in document "Web" changed to "web", "Website/Websites" to "website/websites", "web site/web sites" or "Web site/Web site" to "websites" or "website"
- [ ] Everywhere in document "Success Criteria/Criterion" changed to "success criteria/criterion" (which I think we already handled, but may need to adjust in any quoted language where we made normative word substitutions.
- [ ] Anywhere "HTML" is used added markup to have the abbreviation expanded on hover to "HyperText Markup Language"
- [ ] SC 2.4.11 Focus Not Obscured (Minimum) Note 2 added "visually" to the text.
- [ ] 2.5.1 Pointer Gestures, 2.5.2 Pointer Cancellation, 2.5.7 Dragging Movements changed "i.e." to "i.e.," in the notes
- [ ] 2.5.8 Target Size (Minimum) changed "where:" to "when:", changed the definition list to remove ":" on each term, and changed "User agent control" to "User Agent Control"
- [ ] 3.1.2 Language of Parts changed "text." to be [text](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-text)."
- [ ] SC 3.3.8 Accessible Authentication (Minimum) changed "Web site." to "website."
- [ ] 4.1.2 Name, Role, Value changed "states," to "[states](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-states)," 
- [ ] accessibility supported glossary term - changed "Web Technology" to "web technology"
- [ ] "Cognitive function test" glossary term made "cognitive" lowercase, changed "Web sites" to "websites"
- [ ] "Content (Web content) glossary term, lowercased "web"
- [ ] Removed unused, unreferenced glossary term "encloses"
- [ ] Glossary term "non-text content" changed "ASCII Art" to "ASCII art"
- [ ] Glossary term "programmatically determined link content" changed "paragraph, list" to "paragraph, list item"
- [ ] Glossary term "programmatically set" not sure but seems to have some change to "user agents"
- [ ] Glossary term "single pointer" underwent pretty extensive changes. Will need to check if we need to do an interpretation.
- [ ] Glossary term "status message"  added a link to "process"
- [ ] Glossary term "technology (Web content)" changed "Web Technology" to "web technology" and "Web Content Technologies" to web content technologies"
- [ ] Glossary term "used in an unusual or restricted way" changed "WCAG 2.1" to "WCAG 2"

**Comment by ChrisLoiselle** [comment]
@maryjom @pday1 

> The diff file made this analysis easy, so I identified changes of note throughout in the list below. Now we simply have to determine if these things cause any changes within WCAG2ICT interpretation language. Daniel can handle the update of the directly quoted WCAG text.
> 
> * [ ]  Everywhere in document "Web" changed to "web", "Website/Websites" to "website/websites", "web site/web sites" or "Web site/Web site" to "websites" or "website"

I only see the wcag.json file with reference to website - https://github.com/search?q=repo%3Aw3c%2Fwcag2ict%20Website&type=code

> * [ ] [ ]  Everywhere in document "Success Criteria/Criterion" changed to "success criteria/criterion" (which I think we already handled, but may need to adjust in any quoted language where we made normative word substitutions.

I see many instances of both , https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+success+criteria&type=code , we primarily use success criteria , however capitalization version of the same is used. We also have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+success+criterion&type=code, which showcases the same logic.

> * [ ] [ ]  Anywhere "HTML" is used added markup to have the abbreviation expanded on hover to "HyperText Markup Language"

We have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+HTML&type=code , which showcases this. This occurs on [comments-on-definitions-in-wcag-2.2-glossary-in-appendix-a.md] and [wcag.json]

> * [ ] [ ]  SC 2.4.11 Focus Not Obscured (Minimum) Note 2 added "visually" to the text

We have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+2.4.11&type=code . I don't see visually listed in the text.

> * [ ] .[ ]  2.5.1 Pointer Gestures, 2.5.2 Pointer Cancellation, 2.5.7 Dragging Movements changed "i.e." to "i.e.," in the notes

This needs to be updated per https://github.com/search?q=repo%3Aw3c%2Fwcag2ict%202.5.1&type=code , https://github.com/search?q=repo%3Aw3c%2Fwcag2ict%202.5.2&type=code and https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+2.5.7&type=code

> * [ ] [ ]  2.5.8 Target Size (Minimum) changed "where:" to "when:", changed the definition list to remove ":" on each term, and changed "User agent control" to "User Agent Control"

I don't see the where to when change when reviewing https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+2.5.8&type=code

I don't see the user agent control reference when reviewing  https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+2.5.8+user+agent+control&type=code 

> * [ ] [ ]  3.1.2 Language of Parts changed "text." to be [text](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-text)."

I see this as a needed change per https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+3.1.2+text&type=code

> * [ ] [ ]  SC 3.3.8 Accessible Authentication (Minimum) changed "Web site." to "website."

I see this as a needed change per https://github.com/w3c/wcag2ict/blob/66f2cba067c131176369a9a77c64f950c846681e/comments-by-guideline-and-success-criterion.md?plain=1#L1039

> * [ ] > * [ ] [ ]  4.1.2 Name, Role, Value changed "states," to "[states](https://www.w3.org/TR/2024/REC-WCAG22-20241212/#dfn-states),"

I see this as a needed change per https://github.com/search?q=repo%3Aw3c%2Fwcag2ict%204.1.2&type=code

> * [ ] [ ]  accessibility supported glossary term - changed "Web Technology" to "web technology"[ ]  "Cognitive function test" glossary term made "cognitive" lowercase, changed "Web sites" to "websites"[ ]  "Content (Web content) glossary term, lowercased "web"

I see this as a needed change per https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+Web+Technology&type=code and [ ](https://github.com/w3c/wcag2ict/blob/66f2cba067c131176369a9a77c64f950c846681e/comments-on-definitions-in-wcag-2.2-glossary-in-appendix-a.md?plain=1#L10) 

Per  https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+cognitive+function+test&type=code , cognitive function test is already in lowercase

> * [ ] [ ]  Removed unused, unreferenced glossary term "encloses"[ ]  Glossary term "non-text content" changed "ASCII Art" to "ASCII art"

This term of "encloses" is referenced in https://github.com/search?q=repo%3Aw3c%2Fwcag2ict%20encloses&type=code, I'm not sure if you are asking to remove this or if to check if it is removed. It is present as a term in our comments on definitions if that is the ask here.

Non text content is listed in 6 files, see https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+non-text+content&type=code, so if you are asking to change that, yes, we would need to. The term itself is lower case, however also used in SC as Non-text Contrast. So is dependent on context.

For ASCII art, we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+ASCII+Art&type=code, which references ASCII art in the comments on definitions file.

> * [ ] [ ]  Glossary term "programmatically determined link content" changed "paragraph, list" to "paragraph, list item"[ ]  Glossary term "programmatically set" not sure but seems to have some change to "user agents"[ ]  Glossary term "single pointer" underwent pretty extensive changes. Will need to check if we need to do an interpretation.[ ]  Glossary term "status message"  added a link to "process"

For programmatically determined link text is in lower case, per https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+programmatically+determined+link+content&type=code

For paragraph, list , we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+paragraph%2C+list&type=code which does showcase that this needs to be changed.

For programmatically set , we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+programmatically+set&type=code which is lowercase. 

For user agent, we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+user+agents&type=code which shows user agent and [user agent] use. I'm not sure what the ask is here exactly. 

For single pointer, we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+single+pointer&type=code , so if we are referencing old data, yes, we would want to update. Currently, we reference https://github.com/w3c/wcag2ict/blob/66f2cba067c131176369a9a77c64f950c846681e/comments-on-definitions-in-wcag-2.2-glossary-in-appendix-a.md?plain=1#L61 for glossary items that apply to all technologies. So if we are interpreting, we need to add a glossary term and define it in context.

For status message, this is represented in cap and lowercase, in context https://github.com/search?q=repo%3Aw3c%2Fwcag2ict%20status%20message&type=code. The plural "status messages" is also used.

For process, we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+process&type=code as where we reference this word. I'd need more context as to what should be linked and to where, if we are to update.

> * [ ] [ ]  Glossary term "technology (Web content)" changed "Web Technology" to "web technology" and "Web Content Technologies" to web content technologies"

For technology web content , we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+technology+%28Web+content%29&type=code , where we replace web content technology with non-web document or software... this is a variation on what is referenced in the phrasing asked about here in this checkbox list item. 

For Web Content Technologies, we have https://github.com/search?q=repo%3Aw3c%2Fwcag2ict+Web+Content+Technologies&type=code but don't seem to call out that specific term as per the checkbox list item in this issue 580.

> * [ ] [ ]  Glossary term "used in an unusual or restricted way" changed "WCAG 2.1" to "WCAG 2"

**Comment by pday1** [comment]
@ChrisLoiselle @maryjom I added a few additions to Chris's comment above

**Comment by maryjom** [comment]
A lot of the pointed out "Success Criterion" are either section headings (which in W3C style is capitalized on each word) or in link references to section headings (which I assume would retain the capitalization of the section name).

**Comment by maryjom** [comment]
Analysis is done and the changes have been incorporated.
